js/components/portal-dashboard/all-responses-popup/selected-student-list.tsx

```diff
@@ -0,0 +1,115 @@
+import React from "react";
+import { Map } from "immutable";
+import { AnonymizeStudents } from "../anonymize-students";
+import { QuestionTypes } from "../../../util/question-utils";
+import Answer from "../../../containers/portal-dashboard/answer";
+import { getFormattedStudentName } from "../../../util/student-utils";
+import striptags from "striptags";
+
+import SpotlightIcon from "../../../../img/svg-icons/spotlight-icon.svg";
+import AssignmentIcon from "../../../../img/svg-icons/assignment-icon.svg";
+import SmallCloseIcon from "../../../../img/svg-icons/small-close-icon.svg";
+
+import css from "../../../../css/portal-dashboard/all-responses-popup/selected-student-list.less";
+
+interface IProps {
+  currentActivity?: Map<string, any>;
+  currentQuestion?: Map<string, any>;
+  students?: any;
+  anonymous: boolean;
+  isAnonymous: boolean;
+  handleCloseSpotlightStudentListDialog: (show: boolean) => void;
+  setAnonymous: (value: boolean) => void;
+}
+export class SpotlightStudentListDialog extends React.PureComponent<IProps>{
+  render() {
+    return (
+      <div className={css.spotlightListDialog} data-cy="spotlight-student-list-dialog">
+        {this.renderSpotlightListHeader()}
+        {this.renderColumnHeaders()}
+        {this.renderStudentAnswers()}
+      </div>
+    );
+  }
+
+  private renderSpotlightListHeader = () => {
+    const { currentActivity } = this.props;
+    const activityName = currentActivity?.get("name");
+    return (
+      <div className={css.spotlightListHeader}>
+        <div className={css.headerLeft}>
+          <SpotlightIcon className={css.spotlightIcon} />
+        </div>
+        <div className={css.headerCenter}>
+          <div className={css.assignmentContainer}>
+            <AssignmentIcon className={`${css.assignmentIcon} ${css.icon}`} />
+            <div className={css.title} data-cy="popup-header-title">{activityName}</div>
+          </div>
+        </div>
+        <div className={css.headerRight}>
+          <div className={css.closeIcon} data-cy="close-popup-button" onClick={this.handleCloseDialogClick}>
```

Comment:
With currying in `StudentResponsePopup`, we can write it simply as: `<div className={css.closeIcon} data-cy="close-popup-button" onClick={this.props.onCloseDialog}>`